.github/workflows/scale-test.yaml

```diff
@@ -17,6 +17,14 @@ on:
       image_tag:
         description: "Image Tag (if not set, default for this commit will be used)"
         type: string
+      control_plane:
+        description: "Control Plane"
+        type: choice
+        required: true
+        options:
+          - "legacy"
+          - "hubble"
+        default: "legacy"
```

Comment:
Do we want to leave the default option? It might be good to explicitly mention what control plane we are testing for each test.

Reply:
that's a good point